install_config/persistent_storage/persistent_storage_local.adoc

```diff
@@ -0,0 +1,131 @@
+[[install-config-persistent-storage-persistent-storage-local]]
+= Persistent Storage Using Local Volume
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+{product-title} clusters can be provisioned with xref:../../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[persistent
+storage] by using local volumes. Local persistent volume allows you to access local storage devices such as a disk, partition or directory through the standard PVC interface. An external static xref:../../install_config/configuring_local.adoc#local-volume-config-provisioner[provisioner] and a related xref:../../install_config/configuring_local.adoc#local-volume-config-bootstrap[bootstrapper] are available to simplify local volumes management after they are configured.
+
+Local volumes can be used without manually scheduling pods to nodes, because the system is aware of the volume’s node constraints. However, local volumes are still subject to the availability of the underlying node and are not suitable for all applications.
+
+[NOTE]
+====
+. Local volume is an alpha feature and may change in a future release of {product-title}.
+. The local volume provisioner is different from most provisioners and does not support dynamic provisioning.
+. Local volume provisioner requires that the administrators preconfigure the local volumes on each node and mount them under discovery directories. The provisioner then manages the volumes by creating and cleaning up PersistentVolumes for each volume.
+====
+
+[WARNING]
+====
+Local volumes can only be used as a statically created Persistent Volume.
+====
+
+[[local-volume-create-local-persistent-volume]]
+== Create Local Persistent Volumes
+Create the local volume manually or by bootstraping the external provisioner.
+
+[NOTE]
+====
+You must already have a xref:../../install_config/configuring_local.adoc#local-volume-starting-cluster-with-local-disks[cluster with local disks] configured.
+====
+
+[[local-volume-bootstrap-static-provisioner]]
+=== Option 1: Bootstrap the external static provisioner
+This is optional, only for automated creation and cleanup of local volumes. See xref:../../install_config/configuring_local.adoc#local-volume-config-bootstrap[Local Volume Provisioner Bootstrap] and xref:../../install_config/configuring_local.adoc#local-volume-config-provisioner[Local Volume Provisioner] for details and sample configuration files.
+
+. Create an admin account with cluster admin privilege:
++
+[source,bash]
+----
+oc create -f bootstrapper/deployment/openshift/admin-account.yaml
```

Comment:
where is bootstrapper/deployment/openshift/admin-account.yaml file located? Should we ship it with OpenShift? Should we use the upstream one for alpha features?
cc @childsb